2014-09-29 15:27:45	FAILED	error executing sourceSQL (SELECT _DimAccount.AccountId,ar_aged_debtor_balance.update_datetime,ar_aged_debtor_balance.current_period,ar_aged_debtor_balance.days_30,ar_aged_debtor_balance.days_60,ar_aged_debtor_balance.days_90,ar_aged_debtor_balance.days_90_plus FROM lumo.csv_FactMarketingCampaignActivity INNER JOIN staticdims.DimCustomer as _DimCustomer ON _DimCustomer.CustomerKey = csv_FactMarketingCampaignActivity.CustomerKey AND _DimCustomer.Meta_IsCurrent = 1): AssertionError: expected [RequestError: The multi-part identifier "ar_aged_debtor_balance.days_90_plus" could not be bound.] to not exist
2014-09-29 15:27:45	FAILED	error executing sourceSQL (SELECT _DimAccount.AccountId,ar_aged_debtor_balance.update_datetime,ar_aged_debtor_balance.current_period,ar_aged_debtor_balance.days_30,ar_aged_debtor_balance.days_60,ar_aged_debtor_balance.days_90,ar_aged_debtor_balance.days_90_plus FROM lumo.csv_FactMarketingCampaignActivity INNER JOIN staticdims.DimCustomer as _DimCustomer ON _DimCustomer.CustomerKey = csv_FactMarketingCampaignActivity.CustomerKey AND _DimCustomer.Meta_IsCurrent = 1 AND MarketingCampaignId = '1'): AssertionError: expected [RequestError: The multi-part identifier "ar_aged_debtor_balance.days_90_plus" could not be bound.] to not exist
2014-09-29 15:27:46	FAILED	error executing sourceSQL (SELECT _DimAccount.AccountId,ar_aged_debtor_balance.update_datetime,ar_aged_debtor_balance.current_period,ar_aged_debtor_balance.days_30,ar_aged_debtor_balance.days_60,ar_aged_debtor_balance.days_90,ar_aged_debtor_balance.days_90_plus FROM lumo.csv_FactMarketingCampaignActivity INNER JOIN staticdims.DimCustomer as _DimCustomer ON _DimCustomer.CustomerKey = csv_FactMarketingCampaignActivity.CustomerKey AND _DimCustomer.Meta_IsCurrent = 1 AND MarketingCampaignId = '1'): AssertionError: expected [RequestError: The multi-part identifier "ar_aged_debtor_balance.days_90_plus" could not be bound.] to not exist
2014-09-29 15:27:46	FAILED	error executing sourceSQL (SELECT _DimAccount.AccountId,ar_aged_debtor_balance.update_datetime,ar_aged_debtor_balance.current_period,ar_aged_debtor_balance.days_30,ar_aged_debtor_balance.days_60,ar_aged_debtor_balance.days_90,ar_aged_debtor_balance.days_90_plus FROM lumo.csv_FactMarketingCampaignActivity INNER JOIN staticdims.DimCustomer as _DimCustomer ON _DimCustomer.CustomerKey = csv_FactMarketingCampaignActivity.CustomerKey AND _DimCustomer.Meta_IsCurrent = 1 AND MarketingCampaignId = '1'): AssertionError: expected [RequestError: The multi-part identifier "ar_aged_debtor_balance.days_90_plus" could not be bound.] to not exist
2014-09-29 15:27:46	FAILED	error executing sourceSQL (SELECT _DimAccount.AccountId,ar_aged_debtor_balance.update_datetime,ar_aged_debtor_balance.current_period,ar_aged_debtor_balance.days_30,ar_aged_debtor_balance.days_60,ar_aged_debtor_balance.days_90,ar_aged_debtor_balance.days_90_plus FROM lumo.csv_FactMarketingCampaignActivity INNER JOIN staticdims.DimCustomer as _DimCustomer ON _DimCustomer.CustomerKey = csv_FactMarketingCampaignActivity.CustomerKey AND _DimCustomer.Meta_IsCurrent = 1 AND MarketingCampaignId = '1'): AssertionError: expected [RequestError: The multi-part identifier "ar_aged_debtor_balance.days_90_plus" could not be bound.] to not exist
2014-09-29 15:27:47	FAILED	error executing sourceSQL (SELECT _DimAccount.AccountId,ar_aged_debtor_balance.update_datetime,ar_aged_debtor_balance.current_period,ar_aged_debtor_balance.days_30,ar_aged_debtor_balance.days_60,ar_aged_debtor_balance.days_90,ar_aged_debtor_balance.days_90_plus FROM lumo.csv_FactMarketingCampaignActivity INNER JOIN staticdims.DimCustomer as _DimCustomer ON _DimCustomer.CustomerKey = csv_FactMarketingCampaignActivity.CustomerKey AND _DimCustomer.Meta_IsCurrent = 1 AND MarketingCampaignId = '1'): AssertionError: expected [RequestError: The multi-part identifier "ar_aged_debtor_balance.days_90_plus" could not be bound.] to not exist
2014-09-29 15:27:47	FAILED	error executing sourceSQL (SELECT _DimAccount.AccountId,ar_aged_debtor_balance.update_datetime,ar_aged_debtor_balance.current_period,ar_aged_debtor_balance.days_30,ar_aged_debtor_balance.days_60,ar_aged_debtor_balance.days_90,ar_aged_debtor_balance.days_90_plus FROM lumo.csv_FactMarketingCampaignActivity INNER JOIN staticdims.DimCustomer as _DimCustomer ON _DimCustomer.CustomerKey = csv_FactMarketingCampaignActivity.CustomerKey AND _DimCustomer.Meta_IsCurrent = 1 AND MarketingCampaignId = '1'): AssertionError: expected [RequestError: The multi-part identifier "ar_aged_debtor_balance.days_90_plus" could not be bound.] to not exist
2014-09-29 15:29:49	FAILED	error executing sourceSQL (SELECT _DimAccount.AccountId,ar_aged_debtor_balance.update_datetime,ar_aged_debtor_balance.current_period,ar_aged_debtor_balance.days_30,ar_aged_debtor_balance.days_60,ar_aged_debtor_balance.days_90,ar_aged_debtor_balance.days_90_plus FROM /* Staging */ lumo.ar_aged_debtor_balance INNER JOIN /* Dimensional */ staticDims.DimAccount as _DimAccount ON _DimAccount.AccountKey = ar_aged_debtor_balance.seq_party_id AND _DimAccount.Meta_IsCurrent = 1 AND site_id = '696059'): AssertionError: expected [RequestError: Invalid column name 'site_id'.] to not exist
